lib/portage/dep/__init__.py

```diff
@@ -125,25 +125,24 @@ def _get_atom_re(eapi_attrs: portage.eapi._eapi_attrs) -> re.Pattern:
     cpv_re = _cpv
 
     _atom_re = re.compile(
-        "^(?P<without_use>(?:"
-        + "(?P<op>"
-        + _op
-        + cpv_re
-        + ")|"
-        + "(?P<star>="
-        + cpv_re
-        + r"\*)|"
-        + "(?P<simple>"
-        + cp_re
-        + "))"
-        + "("
-        + _slot_separator
-        + _slot_loose
-        + ")?"
-        + _repo
-        + ")("
-        + _use
-        + ")?\\Z",
+        "^"
+        + (
+            "(?P<without_use>"
+            + (
```

Comment:
It is mysterious to me, that you think this is more readable than, say,

```python
rf"""
    ^
        (?P<without_use>
            (?:
                (?P<op>{_op}{cpv_re})
                |
                (?P<star>={cpv_re}\*)
                |
                (?P<simple>{cp_re})
            )

        ({_slot_separator}{_slot_loose})?
        {_repo}
        )
    ({_use})?
    \Z"""

Reply:
Mysteriously, the code already used re.VERBOSE but never bothered to make use of it??? :D